drivers/storage/portworx/util/csi_generator.go

```diff
@@ -203,9 +203,12 @@ func (g *CSIGenerator) GetCSIConfiguration() *CSIConfiguration {
 	}
 
 	// IncludeExternalHealthMonitor only with PX 2.10.0+ and k8s 1.21+
-	if g.kubeVersion.GreaterThanOrEqual(k8sVer1_21) && g.pxVersion.GreaterThanOrEqual(pxVer2_10) {
```

Comment:
Can we instead keep this block, but set to false with a comment above?

Reply:
done